float64 lat
float64 long
bool under_water